src/features/exporter/Downloader.js

```diff
@@ -153,20 +154,28 @@ class Downloader extends Component {
               index={1}
               model={this.props} />
 
+            {this.props.fileType === 'Theta Rho (.thr)' && <InputOption
```

Comment:
Should we just always multiply by 0.99? Is that going to mess people up?

Reply:
Almost all patterns work fine. Some don't (on my table). I wouldn't change the default from 1 for sure. If you think this feature feels awkward, we can hide it so that I can use it when needed for personal use. Sisyphus forums show this happens to others, but their reply is usually along the lines of "don't move the ball so fast, the tracking can get off". I have not found this to be true in all cases, but I don't know the cause.